Makefile

```diff
@@ -16,3 +16,27 @@ protogen:
 	protoc \
 		-I sdk -I vendor -I vendor/github.com/gogo/protobuf/protobuf \
 		--gogofaster_out=plugins=grpc:pb sdk/*.proto
+
+integration-test: dummy-build build-lookout
+	$(BIN_PATH)/dummy serve &>/dev/null & \
+	PID=$$!; \
+	$(BIN_PATH)/lookout review ipv4://localhost:10302 | grep "BEGIN RESULT"; \
+	if [ $$? != 0 ] ; then \
+		echo "review test failed"; \
+		kill $$PID; \
+		exit 1; \
+	fi; \
+	$(BIN_PATH)/lookout push ipv4://localhost:10302 | grep "dummy comment for push event"; \
+	if [ $$? != 0 ] ; then \
+		echo "push test failed"; \
+		kill $$PID; \
+		exit 1; \
+	fi; \
+	kill $$PID || true ; \
```

Comment:
Scenario looks nice.

But can we please extract this logic to a small .sh file that follows https://google.github.io/styleguide/shell.xml and just call it from this target?

Maintaining Makefiles is a nightmare.

Reply:
this is shell script but inside makefile. Absolutely similar to how src-d CI works.

Reply:
👍 let's extract it to a separate file

Reply:
looks like it's the only sh script we have for now. It's only 15 lines of code, moving it to separate sh would make it more complex. (need to handle lack of binaries for example). Let's keep it here for now.
If we will need to make testing more complex we will move code to .sh files or use some other tools.

Reply:
Makes sense, agreed! Thank you for thoughtful consideration